src/components/dataDisplays/HoustonSightingsDisplay.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+import ActionIcon from '../ActionIcon';
+import { cellRendererTypes } from '../dataDisplays/cellRenderers';
+import DataDisplay from './DataDisplay';
+
+export default function HoustonSightingsDisplay({
+  sightings,
+  columns = ['date', 'location', 'actions'],
+  onDelete,
+  linkPath = 'sightings',
+  noSightingsMsg = 'NO_SIGHTINGS',
+  loading,
+  ...rest
+}) {
+  const allColumns = [
+    {
+      reference: 'date',
+      name: 'time',
+      labelId: 'DATE_OF_SIGHTING',
+      options: {
+        cellRenderer: cellRendererTypes.specifiedTime,
+      },
+    },
+    {
+      reference: 'location',
+      name: 'formattedLocation',
+      labelId: 'LOCATION',
+      options: {
+        cellRenderer: cellRendererTypes.location,
+      },
+    },
+    {
+      reference: 'locationIdValue',
+      name: 'locationId_value',
+      labelId: 'LOCATION',
+    },
+    {
+      reference: 'created',
+      name: 'created',
+      labelId: 'DATE_REPORTED',
+      options: {
+        cellRenderer: cellRendererTypes.date,
+      },
+    },
+    {
+      reference: 'submissionTime',
+      name: 'submissionTime',
+      labelId: 'DATE_REPORTED',
```

Comment:
Are two columns with the same `labelId` assumed to never be used at the same time?

Reply:
correct. one or the other